compiler/passes/convert-uast.cpp

```diff
@@ -330,7 +330,7 @@ struct Converter {
           // Handles case: 'import foo as bar'
           if (auto as = vc->symbol()->toAs()) {
             Expr* mod = convertAST(as->symbol());
-            const char* rename = as->rename()->name().c_str();
+            const char* rename = astr(as->rename()->name().c_str());
```

Comment:
Recommend `ast(ast->rename()->name()` since we have an `astr` overload accepting a `UniqueString`.

Reply:
(also in the other changes in this PR).

I think that the "right" way to fix this is for the old AST builders & constructors to work with UniqueString. At one point I started to look at that but I concluded it was too much of a side issue.